lucille-core/src/test/java/com/kmwllc/lucille/core/FileToFileIngestIT.java

```diff
@@ -13,9 +13,12 @@ public class FileToFileIngestIT {
   @Test
   public void fullIT() throws Exception {
     File f = null;
+    File directory = null;
     try {
       Config config = ConfigFactory.parseFile(new File("src/test/resources/FileToFileIngestIT/file-to-file-example.conf"));
-      f = new File("output/dest.csv");
+      directory = new File("tempIntegrationTestOutput/");
+      directory.mkdir();
+      f = new File("tempIntegrationTestOutput/dest.csv");
```

Comment:
I think the `file-to-file-example.conf` will need to be updated tho this new tempIntegrationTestOutput dir as well